vmdb/app/models/vm_or_template.rb

```diff
@@ -1064,7 +1064,7 @@ def storage2active_proxies(all_proxy_list = nil)
     # MiqServer coresident proxy needs to contact the host and provide credentials.
     # Remove any MiqServer instances if we do not have credentials
     rsc = self.scan_via_ems? ? self.ext_management_system : self.host
-    proxies.delete_if {|p| MiqServer === p} if rsc && rsc.authentication_invalid?
+    proxies.delete_if {|p| MiqServer === p} if rsc && rsc.missing_credentials?
```

Comment:
Again, when scanning(fleecing) a vm, do we really want to try a MiqProxy that is unreachable, has a bad password?